test/snippet/simd/detail/is_ume_simd.cpp

```diff
@@ -1,7 +1,11 @@
 #include <seqan3/simd/detail/ume_simd.hpp>
 using namespace seqan3::detail;
 
+#if __has_include(<umesimd/UMESimd.h>)
+
 using int8x16_t = ume_simd<int8_t, 16>::type;
```

Comment:
If this is in the `<umesimdu/UMESimd.h>` header don't you need to include it ?

Reply:
It will be auto included in `seqan3/simd/detail/ume_simd.hpp`.